WarehouseConfiguration warehouse_configuration
RobotConfiguration[] robot_configurations
---
bool success
